src/components/LightMode.tsx

```diff
@@ -0,0 +1,6 @@
+import { FC, ReactNode } from 'react'
+import { LightMode as OrigLightMode } from '@ironfish/ui-kit'
+
+const LightMode: FC<{ children: ReactNode }> = OrigLightMode
+
+export default LightMode
```

Comment:
Need move it into ui-kit.

Reply:
Seems that I have some node_module react types conflict, fixed.